prometheus/base/alerting/prometheus.controller_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-controller-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: controller_alerts
+    rules:
+    - alert: ControllerReconciliationErrors
+      expr: |
```

Comment:
The requirement as I understand it from @amisstea's document was involving a lookback of 2m window. I don't see this here.

https://docs.google.com/document/d/1akkvRQRiQud-ba644bSrqhpq1asJ4H-YO_DA58BjE-s/edit#heading=h.20eg8stzvxhy

Reply:
I need to check with @amisstea on this since it's not alerting if there is no change in values and error is still there.

Reply:
I think the lookback window might be important or else this could end up detecting old controller errors and these alerts should only consider recent activity.

Reply:
@amisstea Thanks for the clarification. I have updated now.